software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/mongodb/sharding/MongoDBRouterClusterImpl.java

```diff
@@ -56,12 +60,17 @@ public void start(Collection<? extends Location> locations) {
     
     public static class MemberTrackingPolicy extends AbstractMembershipTrackingPolicy {
         @Override protected void onEntityEvent(EventType type, Entity member) {
+            defaultHighlightAction(type, entity);
```

Comment:
Not needed as this policy extends `AbstractMembershipTrackingPolicy ` and `onEntityEvent` [already call `defaultHighlightAction`](https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/entity/group/AbstractMembershipTrackingPolicy.java#L242)

Reply:
this method overrides that method and doesn't call `super` so it has to redo it here, doesn't it?

Reply:
Meh, you're right, I didn't see the missing `super` call for some reason.